apps/native/app/src/screens/wallet-pass/wallet-pass.tsx

```diff
@@ -23,20 +29,52 @@ import {
   useGetLicenseQuery,
 } from '../../graphql/types/schema'
 import { createNavigationOptionHooks } from '../../hooks/create-navigation-option-hooks'
+import { isAndroid, isIos } from '../../utils/devices'
+import { screenWidth } from '../../utils/dimensions'
 import { FieldRender } from './components/field-render'
 
-const Information = styled.ScrollView`
+const INFORMATION_BASE_TOP_SPACING = 70
```

Comment:
Will this work across different devices? I'm thinking iPhone SE

Reply:
Uhh good point I will verify

Reply:
Although it might have already been broken though since this constant was used before this pr
https://github.com/island-is/island.is/blob/154175cd72e39d15655e863960888bc89056e5f4/apps/native/app/src/screens/wallet-pass/wallet-pass.tsx#L36

Reply:
Verified on Simulator at least
![image](https://github.com/island-is/island.is/assets/112904566/004c3a63-7308-4127-ac38-9bab37dd2d05)